tests/setup-softhsm.sh

```diff
@@ -335,6 +335,7 @@ export OPENSSL_CONF="${OPENSSL_CONF}"
 export SOFTHSM2_CONF=${BASEDIR}/${TMPPDIR}/softhsm.conf
 export TESTSSRCDIR="${TESTSSRCDIR}"
 export TESTBLDDIR="${TESTBLDDIR}"
+export PORT_INCREMENT=10
```

Comment:
Why do we need this?
Are you planning to run tests in parallel?

Reply:
It started as part of debugging the issues as the tests kept the processes running as everything was significantly slow (especially under valgrind). It can be probably dropped though. Let me give it a try.

Reply:
Removed. Locally works ok. Lets see the CI.